frontend/app/ui/account/login/route.js

```diff
@@ -1,6 +1,13 @@
+import { inject as service } from "@ember/service";
 import BaseRoute from "customer-center/routes/-base";
 
 export default class AccountLoginRoute extends BaseRoute {
+  @service session;
+
+  beforeModel(transition) {
+    this.session.prohibitAuthentication("index");
+  }
```

Comment:
Shouldn't this route just extend the authenticated base route?

Reply:
This route must not require authentication. Otherwise, the user cannot log in.

Reply:
Aaah so this is basically the ex. unauthenticated route?

Reply:
Yes